extension/src/cli/dvc/runner.ts

```diff
@@ -170,21 +183,40 @@ export class DvcRunner extends Disposable implements ICli {
     return process
   }
 
-  private getOptions(cwd: string, args: Args) {
-    return getOptions(
+  private getOptions(
+    cwd: string,
+    args: Args,
+    studioAccessToken: string | undefined
+  ) {
+    const options = getOptions(
       this.config.getPythonBinPath(),
       this.config.getCliPath(),
       cwd,
       ...args
     )
+
+    if (!studioAccessToken) {
+      return options
+    }
+
+    return {
+      ...options,
+      env: { ...options.env, STUDIO_TOKEN: studioAccessToken }
+    }
   }
 
-  private startProcess(cwd: string, args: Args) {
+  private async startProcess(cwd: string, args: Args) {
     this.pseudoTerminal.setBlocked(true)
     this.processOutput.fire(`Running: dvc ${args.join(' ')}\r\n\n`)
+
+    const studioAccessToken = this.config.sendLiveToStudio()
```

Comment:
[I] Move `sendLiveToStudio` out of the config and into connect. Also make the token getter sync

Reply:
means it can be moved down into `getOptions`